src/Controller/Component/RequestHandlerComponent.php

```diff
@@ -300,6 +300,7 @@ public function beforeRedirect(Event $event, $url, Response $response)
                 'REQUEST_METHOD' => 'GET'
             ]
         ]));
+        $response->statusCode(200);
         $response->send();
         $response->stop();
     }
```

Comment:
Oh no. That would make my [AJAX Component](https://github.com/dereuromark/cakephp-ajax/blob/master/src/Controller/Component/AjaxComponent.php#L100) obsolete :trollface: ^^